Doc/library/inspect.rst

```diff
@@ -702,6 +699,12 @@ function.
       Python.  For example, in CPython, some built-in functions defined in
       C provide no metadata about their arguments.
 
+   .. note::
+
+      We may use the :attr:`!__signature__` attribute to create the signature.
```

Comment:
```suggestion
      If the passed object has a :attr:`!__signature__` attribute, we may use it to create the signature.
```
A little more concrete (your text makes me think "attribute of what?").

Reply:
I agree, that's better.